### PR TITLE
The Alistair Henshaw

### PR DESCRIPTION
**Pull request by Alistair1080**

### CODE REVIEW

File: twitter.html [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    </head>
+    <body>
+        <div class="layout">
+            <aside class="left">
+                <img src="./icons/twitter-app-icon.svg" alt="twitter logo" class="logo">
+                <nav class="nav">
+                    <a href="#main" class="hom bar" tabindex="1">Home</a>
+                    <a href="explore.html" class="exp bar" tabindex="2">Explore</a>
+                    <a href="notifications.html" class="not bar" tabindex="3">Notifications</a>
+                    <a href="messages.html" class="mes bar" tabindex="4">Messages</a>
```
Comment (timmyctt) [inline twitter.html:15]: Have not seen tableindex before. What does it do?

Comment (carolarruda) [reply twitter.html:15]: Yes, spot on Timmy. What does that do?

File: profile.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Profile</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <h1>
+            Looking fresh. The world needs more of your mirror selfies.
```
Comment (carolarruda) [inline profile.html:9]: These were funny. Had fun exploring the page and finding the gems

File: twitter.html (hunk 1 above)
Comment (Alistair1080) [reply twitter.html:15]: elements which aren't normally able to be selected using the tab bar, can be given this so when it gains the ability to be navigated to on the browser using tab.

File: profile.html (hunk 2 above)
Comment (Alistair1080) [reply profile.html:9]: haha thanks, helped me enjoy the challenge more

**Pull request closed without merging**